g11n-ws/vip-common/src/main/java/com/vmware/vip/common/constants/ConstantsKeys.java

```diff
@@ -106,6 +106,7 @@ public class ConstantsKeys {
    public static final String SCOPE_FILTER_MATCH = "(^[a-zA-Z]+[,-_a-zA-Z0-9]+[a-zA-Z0-9]+$)|(^[\\^\\(]+[,-_a-zA-Z0-9]+[\\)]+$)";
 
    public final static Set<String> SOURCE_FORMAT_LIST = new HashSet<String>(Arrays.asList("STRING", "SVG", "MD", "HTML"));
+   public final static String SOURCE_FORMAT_BASE64 = "BASE64";
```

Comment:
why not move "BASE64" to SOURCE_FORMAT_LIST as one of elements?

Reply:
because we need set sourceformat to (BASE64,STRING), (BASE64,SVG) ... when use the base64 encode in clients , if the BASE64 in SOURCE_FORMAT_LIST, we can't compatible this case